diff --git a/audio/include/system/audio.h b/audio/include/system/audio.h
index e9d6002d..0e63c098 100644
--- a/audio/include/system/audio.h
+++ b/audio/include/system/audio.h
@@ -238,6 +238,7 @@ enum {
     FCC_2 = 2,
     FCC_8 = 8,
     FCC_12 = 12,
+    FCC_13 = 13,
     FCC_24 = 24,
     FCC_26 = 26,
     // FCC_LIMIT is the maximum PCM channel count supported through
@@ -245,7 +246,8 @@ enum {
     //
     // This can be adjusted onto a value such as FCC_12 or FCC_26
     // if the device HAL can support it.  Do not reduce below FCC_8.
-    FCC_LIMIT = FCC_12,
+    // 360RA MPEG-H decoder xuat PCM 13ch (CHANNEL_OUT_13POINT0).
+    FCC_LIMIT = FCC_13,
 };
 
 /* A channel mask per se only defines the presence or absence of a channel, not the order.
