diff --git a/services/core/java/com/android/server/audio/AudioService.java b/services/core/java/com/android/server/audio/AudioService.java
index cad3065678d1..d537498e4c26 100644
--- a/services/core/java/com/android/server/audio/AudioService.java
+++ b/services/core/java/com/android/server/audio/AudioService.java
@@ -1532,6 +1532,9 @@ public class AudioService extends IAudioService.Stub
         mSettings = settings;
 
         mAudioPolicy = audioPolicy;
+        if (SystemProperties.getBoolean("ro.config.disable_audio_hardening", false)) {
+            mHardeningOverride.set(HardeningOverride.DISABLE);
+        }
         mAudioPolicy.registerOnStartTask(() -> {
             mAudioPolicy.setHardeningOverride((byte) mHardeningOverride.get());
         });
