diff --git a/init/init.qti.display_boot.sh b/init/init.qti.display_boot.sh
index dbc9db36bb..0e9f8afc23 100644
--- a/init/init.qti.display_boot.sh
+++ b/init/init.qti.display_boot.sh
@@ -82,7 +82,6 @@ case "$target" in
         setprop vendor.display.enable_latch_media_content 1
         setprop vendor.display.enable_inline_writeback 1
         setprop vendor.display.timed_render_enable 1
-        setprop debug.sf.disable_client_composition_cache 0
         # set property for HHG
         if [ "$subtype_id" -eq 1 ]; then
             setprop vendor.display.disable_system_load_check 1
@@ -108,7 +107,6 @@ case "$target" in
         setprop vendor.display.enable_latch_media_content 1
         setprop vendor.display.enable_inline_writeback 0
         setprop vendor.display.timed_render_enable 1
-        setprop debug.sf.disable_client_composition_cache 0
         setprop vendor.display.enable_qsync_idle 1
         ;;
     esac
